src/scss/components/_navbar.scss

```diff
@@ -4,9 +4,18 @@
             justify-content: center;
             margin-left: auto;
         }
-    
+
         .navbar-end {
             margin-left: 0;
         }
     }
+
+    .navbar-dropdown.is-boxed {
+        visibility: hidden;
```

Comment:
Hi @service-paradis ,

There is actually a cleaner solution: although `transition-property` does not work with `display`, it works with `visibility`. And `visibility: hidden` disables tab event as well.

Therefore, I put `visibility: hidden` as default for `.navbar-dropdown.is-boxed`, and when dropdown is open `.navbar-item.has-dropdown.is-active`, set to `visibility: visible;`.